frontend/app/src/components/ui/badge.tsx

```diff
@@ -14,6 +14,8 @@ const badgeVariants = cva(
         destructive:
           'border-transparent bg-destructive text-destructive-foreground shadow hover:bg-destructive/80',
         outline: 'text-foreground',
+        outlineDestructive:
```

Comment:
nit: I think it's more tailwind-thonic to use `kebab-case` e.g. `outline-destructive` for things like this

Reply:
personal pref to not have kebab in ts keys... not strongly opinionated here

Reply:
>ui nit: do you think we can make the status buttons grow so they're all the same width? 
github is broken since there are two commends on the same line i think.

no, this is just the badge... lets tweak the general layout in a separate pr!